k8s/hammerdb/deployment-hammerdb.yaml

```diff
@@ -25,3 +25,20 @@ spec:
             - "10000"
           ports:
             - containerPort: 8080
+          volumeMounts:
+            - name: vol-mariadb-conf
+              mountPath: /home/hammerdb/HammerDB-4.5/maria_ephemeral_tprocc_build.tcl
+              subPath: maria_ephemeral_tprocc_build.tcl
+            - name: vol-mariadb-conf
```

Comment:
are we able to have 2 volumes with the same name? this volume has the same name as above. Same with the volume on like 38.

Reply:
this is fine as the deployment is different.  I have placed all of the text files into one configmap.

Each of these configuration files is referenced by a different subPath